src/com/owncloud/android/files/InstantUploadBroadcastReceiver.java

```diff
@@ -56,22 +51,26 @@
     @Override
     public void onReceive(Context context, Intent intent) {
         Log_OC.d(TAG, "Received: " + intent.getAction());
-        if (intent.getAction().equals(android.net.ConnectivityManager.CONNECTIVITY_ACTION)) {
```

Comment:
This change is useless. All conditions checking the equality with the same string. The action of the Intent